src/content/api/compilation-hooks.md

```diff
@@ -541,6 +541,8 @@ compilation.hooks.afterOptimizeChunkAssets.tap('MyPlugin', chunks => {
 ```
 
 
+
```

Comment:
remove this extra 2 blank lines please

Reply:
@EugeneHlushko Oh! my bad. Reverted. Thanks.